specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/keyvault.json

```diff
@@ -0,0 +1,1835 @@
+{
```

Comment:
It's difficult to tell what was added or otherwise changed in GitHub this way. Please generate a semantic diff using the following instructions:

1. Go to https://aka.ms/openapihub
1. Click on **Specs Diff & Change Log**.
1. Select **Using OpenAPI Configuration File & Tag**.
1. Set the **Source URL** to the new or changed readme.md file, e.g. <https://github.com/Azure/azure-rest-api-specs/blob/4ccfff0a78bee40837834c3780c2b10d8641bc62/specification/keyvault/resource-manager/readme.md>
1. Select the new version in **Tags from Source URL**, e.g. "package-preview-2020-04".
1. Set the **Target URL** to the old readme.md file, e.g. <https://github.com/Azure/azure-rest-api-specs/blob/master/specification/keyvault/resource-manager/readme.md>
1. Select the old version in **Tags from Target URL**, e.g. "package-2019-09".
1. Click **Compare!**.

You can paste the link here. Note: I might have source and target reversed because the labels are not obvious. I've asked the OpenAPI Hub team to change this to just "old" and "new", for example.